framework/field_functions/field_function_grid_based.h

```diff
@@ -120,7 +115,7 @@ class FieldFunctionGridBased : public FieldFunction
    */
   MakeFieldVector(const SpatialDiscretization& discretization, const UnknownManager& uk_man);
 
-  const BoundingBox local_grid_bounding_box_;
+  const std::pair<Vector3, Vector3> local_grid_bounding_box_;
```

Comment:
There were a few typedefs (and this is one) that I removed because they were only used in one place.  I'm all for improving code readability, so I've reverted this change.